### ISSUE TITLE
Setup not working correctly for SwiftUI based projects using new Xcode versions

### ISSUE THREAD

**Issue by arthurpalves** [Backbase/variants#217]

**Comment by GMinucci** [comment]
I assume this issue is regarding new projects that are SwiftUI based not containing the `info.plist` once the project is just created. 
I noticed that the plist won't exist until the default values are changed, once that is done, Xcode will generate the file in the folder.
I'll move this to the next release as we need to discuss on how to solve this issue. I see a few possible approaches:
- Having a default plist in case not available in the project
- Digging deeper in Xcode to see if the plist exists but hidden somewhere

**Comment by GMinucci** [comment]
Adding a bit more information here, in case we want to create the `info.plist` in case it doesn't exist the effort is a bit more than just creating the file.
More information in this answer here with the steps needed: https://stackoverflow.com/questions/67896404/where-is-info-plist-in-xcode-13-missing-not-inside-project-navigator

**Comment by GMinucci** [comment]
The majority of the cases you need a `info.plist` file so this has a low priority to be fixed.
